pbxproj/pbxsections/PBXVariantGroup.py

```diff
@@ -0,0 +1,30 @@
+from pbxproj.PBXGenericObject import *
+from pbxproj.pbxsections.PBXFileReference import *
+
+
+class PBXVariantGroup(PBXGenericObject):
+    @classmethod
+    def create(cls, name, children=None, source_tree=u'<group>'):
+        return cls().parse({
+            u'_id': cls._generate_id(),
+            u'isa': cls.__name__,
+            u'children': children if children else [],
+            u'name': name,
+            u'sourceTree': source_tree
+        })
+
+    def add_variant(self, file_ref):
+        if not isinstance(file_ref, PBXFileReference):
+            return False
+
+        self.children.append(file_ref.get_id())
+        return True
+
+    def remove_variant(self, file_ref):
+        if not isinstance(file_ref, PBXFileReference):
+            return False
+
+        self.children.remove(file_ref.get_id())
```

Comment:
Identified a potential vulnerability, here’s a suggested change for Attempting to remove a file reference that does not exist in the children list will raise a ValueError.:

```suggestion
        if file_ref.get_id() in self.children:
            self.children.remove(file_ref.get_id())
```